### ISSUE TITLE
Allow for retrieving epubs files?

### ISSUE THREAD

**Issue by hneutr** [pgcorpus/gutenberg#31]
Hello! First, this is an awesome project, and I am absolutely thrilled it exists. 

Second, and the reason for the issue: would it be possible to include the raw epub files in the distribution, or allow an option to download them? I see that in `data/.mirror/cache/epub/` there are files with the schema `{ID}/pg{ID}.txt.utf8` — I'm doing some research that leverages the structure provided by epubs to identify section breaks, and it would be _fantastic_ to use this corpus. 

If this isn't something you have time to do but would be willing to discuss, I'd be interested in working to add this myself.

**Comment by fontclos** [comment]
Hi! Thanks for your kind words, we're also quite excited about the project.

At the moment we do not plan to introduce new features. Our focus is on adding better documentation, unit tests, etc. 

So regarding epub files, I recommend you fork the repository and try to modify the code. Do I understand correctly that you'd need the `.epub` files? If so, you would need to find out where are they stored in the PG mirror and modify the call to `rsync` accordingly. I'd be willing to help if you encounter problems. 

TL;DR: fork us!
